MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -549,7 +557,7 @@ - (IBAction)clearCache:(id)sender
     }
     else
     {
-        _resultView.text = [NSString stringWithFormat:@"Failed to clear cache, error = %d", (int)status];
+        _resultView.text = @"Failed to clear the cache.";
```

Comment:
nit: would be nice to still print the error

Reply:
done